sensor/kubernetes/certrefresh/localscanner/certificate_requester.go

```diff
@@ -6,24 +6,28 @@ import (
 	"github.com/pkg/errors"
 	"github.com/stackrox/rox/generated/internalapi/central"
 	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/sync"
 	"github.com/stackrox/rox/pkg/uuid"
 	"github.com/stackrox/rox/sensor/common/message"
+	"github.com/stackrox/rox/sensor/kubernetes/certrefresh/certrequester"
 )
 
 var (
+	log = logging.LoggerForModule()
+
 	// ErrCertificateRequesterStopped is returned by RequestCertificates when the certificate
 	// requester is not initialized.
-	ErrCertificateRequesterStopped                      = errors.New("stopped")
-	_                              CertificateRequester = (*certificateRequesterImpl)(nil)
+	ErrCertificateRequesterStopped                                    = errors.New("stopped")
```

Comment:
Something like
```suggestion
	ErrCertificateRequesterStopped                                    = errors.New("certificate requester has been stopped")
```
?

Reply:
Maybe `... is stopped` instead? In case it was never started?

Reply:
Done